docs/sources/operators-guide/architecture/components/ruler/index.md

```diff
@@ -10,22 +10,39 @@ weight: 130
 The ruler is an optional component that evaluates PromQL expressions defined in recording and alerting rules.
 Each tenant has a set of recording and alerting rules and can group those rules into namespaces.
 
-[//]: # "Diagram source of ruler interactions https://docs.google.com/presentation/d/1LemaTVqa4Lf_tpql060vVoDGXrthp-Pie_SQL7qwHjc/edit#slide=id.g11658e7e4c6_0_938"
+## Operational modes
 
-![Architecture of Grafana Mimir's ruler component](ruler.svg)
+The ruler supports two different rule evaluation modes:
 
-## Recording rules
+### Internal
+
+This is the default mode. The ruler internally runs a querier and distributor, and evaluates recording and alerting rules in the ruler process itself.
+To evaluate rules, the ruler connects directly to ingesters and store-gateways, and writes any resulting series to the ingesters.
 
-The ruler evaluates the expressions in the recording rules at regular intervals and writes the results back to the ingesters.
-The ruler has a built-in querier that evaluates the PromQL expressions and a built-in distributor, so that it can write directly to the ingesters.
 Configuration of the built-in querier and distributor uses their respective configuration parameters:
 
 - [Querier]({{< relref "../../../configuring/reference-configuration-parameters/index.md#querier" >}})
```

Comment:
this path is invalid

Reply:
you sure? it worked for me. 
first it failed to load but then it reloaded itself and fetched the page.

Reply:
We also validate all links in CI. If a link is broken (in the doc), the CI doesn't pass.

> this path is invalid

Where are you seeing it broken? I would like to further investigate it.